frontend/src/react/Correction/CopyCorrection.js

```diff
@@ -0,0 +1,98 @@
+import React, { Component } from 'react';
+import {getExam } from '../../repository/exams';
+import {getCopy} from '../../repository/copies';
+import DisplayExerciceWithAnswer from './DisplayExerciceWithAnswer';
+import { Link } from 'react-router-dom';
+class CopyCorrection extends Component {
+  state = {
+    exam: false,
+    copy: false,
+  };
+  constructor(props) {
+    super(props);
+    this.addExamToState(this.props.route.match.params.examId);
+    this.addCopyToState(this.props.route.match.params.copyId)
+  }
+  async addExamToState(examId) {
+    const exam = await getExam(examId, true);
+    this.setState({...this.state, exam: exam});
```

Comment:
Why deconstruct the state to put it back ? if you update exam it doesn't delete copy and vice versa